CMakeLists.txt

```diff
@@ -5,7 +5,7 @@ if (NOT CMAKE_MESSAGE_CONTEXT)
     set(CMAKE_MESSAGE_CONTEXT_SHOW ON CACHE BOOL "Show CMake message context")
 endif()
 
-project(streaming_protocol_complete VERSION 1.1.0 LANGUAGES CXX)
+project(streaming_protocol_complete VERSION 1.2.0 LANGUAGES CXX)
```

Comment:
If we do this we need also a release branch 1.2.0 afterwards if we want to do a release tag. If we want to have just on a working master we should use 1.1.1 for example until we say we release it,

Reply:
In this case version number follows the rule - "If we adding features / extensions to the document we are increasing y in the x.y.z.". 
I'll create tag and a release branch 1.2.0